Dockerfile

```diff
@@ -44,12 +50,18 @@ FROM build-image AS build-stage
 ARG REPONAME
 ARG BUILD_PACKAGES
 ARG BUILD_TAR_PACKAGES
+ARG BUILD_PIP_PACKAGES
 
 RUN if [ -n "$BUILD_PACKAGES" ]; then \
         apt update && \
         apt install -y --no-install-recommends $BUILD_PACKAGES; \
     fi
 RUN lnls-get-n-unpack -r $BUILD_TAR_PACKAGES
+RUN if [ -n "$BUILD_PIP_PACKAGES" ]; then \
+        apt update && \
+        apt install -y --no-install-recommends pip && \
+        pip install $BUILD_PIP_PACKAGES; \
+    fi
```

Comment:
Do we have a case where we already know this is necessary? It's adding more code and args which we won't even be testing...